lib/Math/Odd.ark

```diff
@@ -0,0 +1,2 @@
+(let odd (fun (n)

```

Comment:
since odd is using abs and mod, it should also import them

Reply:
I only needed to import `abs`.